server/lib/adapters/mysql.js

```diff
@@ -4,6 +4,7 @@
 import mysql from "mysql";
 import Promise from "bluebird";
 import SequelizeUtils from "sequelize/lib/utils";
+import parseConnectionConfig from "../utils/connstr-mysql";
```

Comment:
yeah, I think those utils should leave inside the `adapters/mysql` namespace. You could create a `adapters/mysql` directory and have the current file in the `adapters/mysql/index.js` and the utils as a sibling file.